### PR TITLE
#85 - Create user management table

### PR DESCRIPTION
**Pull request by Haris188**

### CODE REVIEW

File: web-client/src/shared/components/organization/members/Table.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        {loading
+        ? <CircularProgress />
+        : <TableBody>
+            {members.length > 0 
+            ? members.map(member=>(
+                <TableRow key={member.id}>
+                    <TableCell>{member.name}</TableCell>
+                </TableRow>
+            ))
+            : <Typography>No Member to show</Typography>
```
Comment (l2silver) [inline web-client/src/shared/components/organization/members/Table.tsx:46]: Not sure about p tag in a table. Do you think we could either put this inside a cell, or move it out of the table somehow?

Thanks.

File: web-client/src/shared/redux/organization/members/index.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import {
```
Comment (l2silver) [inline web-client/src/shared/redux/organization/members/index.ts:1]: i love how clean and simple this all is. Great work

**Review by l2silver — CHANGES_REQUESTED** [review]
Awesome work, just one little change. Thanks!

File: web-client/src/shared/components/organization/members/Table.tsx (hunk 1 above)
Comment (Haris188) [reply web-client/src/shared/components/organization/members/Table.tsx:46]: Yes!

**Comment by Haris188** [comment]
@l2silver  Is this good now?

**Review by l2silver — APPROVED** [review]

**Pull request merged**